dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/controller/DPPPTController.java

```diff
@@ -125,6 +128,9 @@ public DPPPTController(DPPPTDataService dataService, String appSource,
 
 		List<Exposee> exposees = new ArrayList<>();
 		for (var exposedKey : exposeeRequests.getExposedKeys()) {
+			if (exposedKey == null) {
```

Comment:
Is there a possibility of `exposedKey` being Null? If not I would rather try to stay concise and not introduce checks which will always evaluate to false.

Reply:
With the json `{"exposedKeys":[null]}` and similar a Null value for `exposedKey` can currently pass validation.
I removed the null check in favor of adjusting the validation to handle this case. Thanks for the suggestion!